internal/remote/auth.go

```
@@ -0,0 +1,442 @@
+				CaptchaToken:            nil,
+				UseAppleBusinessUpdates: false,
+			},
+			AmplitudeDeviceID:  req.AmplitudeDeviceID,
+			AmplitudeSessionID: req.AmplitudeSessionID,
+		},
+	}
+	response, err := client.callablePost(ctx, partifulCallableHost+"/sendAuthCodeTrusted", body)
+	if err != nil {
+		return err
+	}
+	defer response.Body.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `response.Body.Close` is not checked

(errcheck)